src/Aquifer.Public.API/Program.cs

```diff
@@ -48,7 +48,11 @@
         config.Endpoints.Configurator = ep => ep.AllowAnonymous();
     })
     .UseOpenApi()
-    .UseReDoc(options => options.Path = "/docs");
+    .UseReDoc(options =>
+    {
+        options.Path = "/docs";
+        options.DocumentTitle = "Aquifer API Docs";
```

Comment:
Is this removing the (1.0.0) piece? Would prefer "Documentation" over "Docs".

Reply:
It's setting the page title so it shows right in the browser tab bar:
![image](https://github.com/user-attachments/assets/47f00c3c-7d06-4711-b68f-facd4440434b)

Reply:
`Documentation` feels a bit long for the page title, we could also just drop it entirely and do `Aquifer API`. Thoughts?

Reply:
I think it's pretty common for page titles to be that length or longer (I don't have a single tab open right now that isn't as long). If there's a quick and similar way to get rid of that "(1.0.0)" as part of this PR that'd be great.

Reply:
Addressed this